zerver/lib/export.py

```diff
@@ -1069,9 +1086,9 @@ def create_soft_link(source, in_progress=True):
         subprocess.check_call(['rm', '-f', in_progress_link])
         new_target = done_link
 
-    subprocess.check_call(["ln", "-nsf", source, new_target])
+    subprocess.check_call(["ln", "-nsf", str(source), new_target])
     if is_done:
-        logging.info('See %s for output files' % (new_target,))
+        logging.info('See %s for output files' % (str(new_target),))
```

Comment:
why do you need these `str()` wrappers?  I guess I'm wondering if these should be `force_str` from zerver.lib.str_utils, for example.